models/Response.cfc

```diff
@@ -5,7 +5,7 @@
 * ********************************************************************************
 * HTTP Response model, spice up as needed
 */
-component accessors="true" {
+component accessors="true" scope="request" {
```

Comment:
The Wirebox component attribute scoping is pretty cool stuff when you dig in to it.  `session` and `client` are tricky with serialization, but this is the perfect use-case for a "singleton" in the request scope